tests/Feature/Http/Livewire/Validators/MissedBlocksTest.php

```diff
@@ -192,102 +185,29 @@
         ->call('sortBy', 'age')
         ->assertSet('sortDirection', SortDirection::ASC)
         ->assertSeeInOrder([
-            'validator-1',
-            'validator-2',
-            'validator-1',
-            'validator-2',
-        ]);
-});
-
-it('should sort name in ascending order', function () {
-    $wallet1 = Wallet::factory()->activeValidator()->create([
-        'attributes' => [
-            'username'             => 'validator-1',
-            'validatorVoteBalance' => 10000 * 1e18,
-        ],
-    ]);
-
```

Comment:
sort by name is not used anymore, some tests that sort by name were removed

Reply:
Fair point, it was removed when we removed the unimplemented logic (multisig, multipay, username, etc). It can be re-added as and when needed :oleglala: